aries-site/src/examples/templates/forms/SettingsExample.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Main,
+  MaskedInput,
+  RangeInput,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { FormContainer } from '.';
+
+export const SettingsExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    doNotDisturbHours: '9:00pm - 5:00am',
+    notifications: true,
+    notificationsVolume: 0.7,
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Settings
+          </Heading>
+          <Text>for HPE Service</Text>
+        </Header>
+        <Main
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            value={formValues}
+            onChange={setFormValues}
+            onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+          >
+            <Box margin={{ bottom: 'small' }}>
+              <Text margin={{ left: 'small' }}>Key feature</Text>
+              <FormField
```

Comment:
Do we want to adjust the left padding/margin (not sure which it is) of label?

Current:
<img width="383" alt="Screen Shot 2020-03-05 at 9 03 33 AM" src="https://user-images.githubusercontent.com/12522275/76006014-b6b0e700-5ec0-11ea-89c6-17e7a4a5797d.png">

Design:
<img width="404" alt="Screen Shot 2020-03-05 at 9 07 37 AM" src="https://user-images.githubusercontent.com/12522275/76006057-c7615d00-5ec0-11ea-94c6-aa07deccd5db.png">

Reply:
Alignment of Labels is consistent throughout all the examples. This discrepancy between design and theme is an item to be discussed in: https://github.com/hpe-design/design-system/issues/458